specification/machinelearningservices/AzureAI.Assets/routes.tsp

```diff
@@ -120,3 +134,118 @@ interface Indexes {
     PagedIndex
   >;
 }
+
+@added(Versions.`2024-05-01-preview`)
+interface Prompts {
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Waiting for bug fix: https://github.com/Azure/typespec-azure-pr/issues/3739."
+  @doc("Get a specific version of a Prompt.")
+  @route("/prompts/{name}/versions/{version}")
+  @get
+  get is Azure.Core.Foundations.Operation<
+    {
+      @doc("Name of the prompt")
+      @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_]*$")
+      @maxLength(254)
+      @path
+      name: string;
+
+      @doc("Version of the prompt")
+      @path
+      version: string;
```

Comment:
Any doc or API on `{version}` (I seems not finding an API to return all versions)? How does customer know which version to send in path?

Reply:
Yes this is a user set version, it's up to them to define the versioning